src/main/java/duke/test/ParserTest.java

```diff
@@ -0,0 +1,40 @@
+package duke.test;

```

Comment:
hi i think your test is in the wrong directory it should be parallel to main like this 
![image](https://github.com/nus-cs2103-AY2324S2/ip/assets/95204474/d7a63728-91a1-4d80-9dc1-6a4b2536ec09)